src/worldofzuul/Cook.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package worldofzuul;
+
+/**
+ *
+ * @author Rasmus
+ */
+public class Cook extends Character {
+    
+    /**
+     * Names the character "Cook".
+     */
+   private String name = "Cook";
```

Comment:
Think a better solution would be to set the name with a constructor.
```java
public Cook () {
     super("Cook");
}
```

Reply:
Fixed in e5d88f3b36c3505ecf61be143de7681bea18ff7e